plugins/dra-ocp-validator/tests/test-dra-admin-access.sh

```
@@ -0,0 +1,165 @@
+NAMESPACE_AUTH="dra-with-admin"
+
+oc create namespace ${NAMESPACE_AUTH}
+oc label namespace ${NAMESPACE_AUTH} "resource.kubernetes.io/admin-access=true"
+echo "Created namespace WITH admin-access label"
+echo ""
+
+oc get namespace ${NAMESPACE_AUTH} -o yaml | tee ${LOG_DIR}/test2-namespace.yaml | grep -A 10 "labels:"
+echo ""
+
+echo "--- Creating ResourceClaim with adminAccess=true ---"
+cat <<EOF | tee ${LOG_DIR}/test2-claim.yaml | oc apply -f - 2>&1 | tee ${LOG_DIR}/test2-result.txt
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)